lib/Optimizer/CodeGen/ConvertToQIR.cpp

```diff
@@ -204,6 +204,37 @@ void cudaq::opt::initializeTypeConversions(LLVMTypeConverter &typeConverter) {
   cudaq::opt::populateCCTypeConversions(&typeConverter);
 }
 
+std::pair<std::size_t, std::vector<std::size_t>>
```

Comment:
It might be nice to have a Doxygen comment about what exactly is being returned here.

Reply:
will do

Reply:
done!